frontend/src/components/CustomField.vue

```diff
@@ -0,0 +1,174 @@
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item class="px-0">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Path</v-list-item-subtitle>
+          <v-list-item-title>{{path}}</v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item class="px-0">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Default Value</v-list-item-subtitle>
+          <v-list-item-title>
+            <template v-if="defaultValue">
+              {{defaultValue}}
+            </template>
+            <span v-else class="font-weight-light text--disabled">Not defined</span>
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-divider></v-divider>
+      <v-subheader class="px-0">Cluster List Settings</v-subheader>
+      <v-list-item class="px-0">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Property Visible</v-list-item-subtitle>
+          <v-list-item-title>
+              {{showColumn ? 'Yes' : 'No'}}
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item class="px-0" v-if="showColumn">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Property Visible by Default</v-list-item-subtitle>
+          <v-list-item-title>
+              {{columnSelectedByDefault ? 'Yes' : 'No'}}
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item class="px-0" v-if="showColumn">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Weight</v-list-item-subtitle>
+          <v-list-item-title>
+            <template v-if="weight">
+              {{weight}}
+            </template>
+            <span v-else class="font-weight-light text--disabled">Not defined</span>
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item class="px-0">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Searchable</v-list-item-subtitle>
+          <v-list-item-title>
+              {{searchable ? 'Yes' : 'No'}}
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+      <v-divider></v-divider>
+      <v-subheader class="px-0">Cluster Details Settings</v-subheader>
+      <v-list-item class="px-0">
+        <v-list-item-content class="pt-1">
+          <v-list-item-subtitle>Property Visible</v-list-item-subtitle>
+          <v-list-item-title>
+              {{showDetails ? 'Yes' : 'No'}}
+          </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
+  </g-popper>
+</template>
+
+<script>
+
+import GPopper from '@/components/GPopper'
+
+export default {
+  name: 'worker-group',
+  components: {
+    GPopper
+  },
+  props: {
```

Comment:
No required properties and no default values?

Reply:
added the required props (name, path). Don't think any default values need to be set, or do you have something in mind?